src/Microsoft.Health.Fhir.Core/Messages/Get/GetSystemCapabilitiesResponse.cs

```diff
@@ -4,19 +4,20 @@
 // -------------------------------------------------------------------------------------------------
 
 using EnsureThat;
-using Hl7.Fhir.Model;
+using Hl7.Fhir.ElementModel;
+using Microsoft.Health.Fhir.Core.Models;
 
 namespace Microsoft.Health.Fhir.Core.Messages.Get
 {
     public class GetSystemCapabilitiesResponse
     {
-        public GetSystemCapabilitiesResponse(CapabilityStatement capabilityStatement)
+        public GetSystemCapabilitiesResponse(ResourceElement capabilityStatement)
```

Comment:
Should we add validation to make sure the InstanceType is correct? E.g., if someone passes in ResourceElement with underlying object being Bundle should be rejected?

Reply:
I don't have a good solution for this atm